<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryEditor</class>
 <widget class="QWidget" name="InventoryEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>312</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string comment="The title of the 'Inventory Editor' tab in the editor.">Inventory Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblSearch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string comment="Search/Add label, for searching/adding items.">Search / Add:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSearchItem">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddItem">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string comment="Item add button." extracomment="Inserts the item into the users item box, in the appropriate section.">Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkAllTabs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string comment="Whether to search all tabs or just the currently selected tab.">Search All Tabs</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabEditors">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cmbSearchItem</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>InventoryEditor</receiver>
   <slot>SearchIndexChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbSearchItem</sender>
   <signal>activated(int)</signal>
   <receiver>InventoryEditor</receiver>
   <slot>SearchIndexChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddItem</sender>
   <signal>clicked()</signal>
   <receiver>InventoryEditor</receiver>
   <slot>ItemAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SearchIndexChange(int)</slot>
  <slot>ItemAdd()</slot>
 </slots>
</ui>
